### PR TITLE
ENH Render some buttons for blocks that can not be edited inline

### PR DESCRIPTION
**Pull request by maxime-rainville**
Some block types can not be edited in an Elemental Area (e.g.: UserForm blocks). Current behaviour is to not render any actions for those blocks in the Elemental Area.

However some action still make sense in that context (e.g.: Duplicate, Archive, Unpublish).

This PR allows those extra actions to be displayed.

# Parent issue
- Fixes https://github.com/silverstripe/silverstripe-elemental/issues/512

### CODE REVIEW

File: client/src/components/ElementEditor/tests/Header-test.js [hunk 1]
```diff
@@ -170,7 +170,7 @@ describe('Header', () => {
         />
       );
 
-      expect(wrapper.text()).not.toContain('ElementActionsComponent');
+      expect(wrapper.text()).toContain('ElementActionsComponent');
```
Comment (maxime-rainville) [inline client/src/components/ElementEditor/tests/Header-test.js:173]: This is shallow rendered, and it won't go all the way to rendering the actual buttons, so I can't make assertions beyond checking that its rendering the action list or not.

**Review by maxime-rainville — COMMENTED** [review]
Turns out we didn't the Context provider non-sense at all. We just needed to pass the expandable prop all the way down.

**Review by emteknetnz — CHANGES_REQUESTED** [review]
Tested locally, does what it should do.  Unpublish button remains on elemental-userforms which makes sense IMO.  Some minor changes requested

File: client/src/components/ElementEditor/ElementActions.js [hunk 2]
```diff
@@ -91,8 +92,8 @@ class ElementActions extends Component {
         dropdownMenuProps={{ right: true }}
         dropdownToggleClassNames={dropdownToggleClassNames}
       >
-        { this.renderEditTabs() }
-        { this.renderDivider() }
+        { expandable && this.renderEditTabs() }
```
[the remark this answers is not in the document]
Comment (emteknetnz) [reply client/src/components/ElementEditor/ElementActions.js:95]: @maxime-rainville remember to do this one

File: client/src/components/ElementEditor/ElementActions.js [hunk 3]
```diff
@@ -59,12 +60,15 @@ class ElementActions extends Component {
    * @returns {DropdownItem|null}
    */
   renderDivider() {
-    const { children, editTabs } = this.props;
+    const { children, editTabs, expandable } = this.props;
 
-    if (editTabs && editTabs.length && React.Children.count(children)) {
```
Comment (maxime-rainville) [inline client/src/components/ElementEditor/ElementActions.js:64]: I swap the return null and return DropdownItem around just so the approach between renderDivider and renderTabs are consistien

**Review by maxime-rainville — COMMENTED** [review]
I was thinking it might be nice to cover this behavior with some behat test, but we can't really do it without requiring userform and the userfrom block because that's the only block that can't be rendered inline.

**Comment by GuySartorelli** [comment]
> we can't really do it without requiring userform and the userfrom block because that's the only block that can't be rendered inline.

I haven't done anything with heat tests yet so excuse my ignorance but I take it then that behat tests can't take advantage of classes that implement `TestOnly`?

**Review by emteknetnz — APPROVED** [review]

**Comment by maxime-rainville** [comment]
> I haven't done anything with behat tests yet so excuse my ignorance but I take it then that behat tests can't take advantage of classes that implement TestOnly?

@GuySartorelli Exactly ... your behat test need to be run against an actual functional website. Occasionally, we'll install frameworktest so we can test specific features that are not directly used in our core modules.

**Comment by emteknetnz** [comment]
@GuySartorelli example behat equivalent of a TestOnly page - https://github.com/silverstripe/silverstripe-frameworktest/blob/4/code/multitab-validation/MultiTabPage.php

**Pull request merged**